### PR TITLE
RecordingOptions: visualize noise suppression and amplification

### PR DESCRIPTION
**Pull request by achimmihca**
### What does this PR do?

- recording options mic visualization considers noise suppression and amplification
- mini refactoring of generic Limit method in NumberUtils

### Closes Issue(s)

#92

### CODE REVIEW

File: UltraStar Play/Assets/Scenes/Options/RecordingOptions/AmplificationSlider.cs [hunk 1]
```diff
@@ -12,7 +12,7 @@ public class AmplificationSlider : TextItemSlider<int>
     protected override void Awake()
     {
         base.Awake();
-        Items = new List<int> { 0, 6, 12, 18 };
+        Items = new List<int> { 0, 3, 6, 9, 12, 15, 18 };
```
Comment (basisbit) [inline UltraStar Play/Assets/Scenes/Options/RecordingOptions/AmplificationSlider.cs]: why did you add more options here? Don't we want to keep it as simple as possible? I don't think the additional steps will improve peoples gameplay because people are not machines and their distance from the microphones varies a bit while playing.

Comment (achimmihca) [reply UltraStar Play/Assets/Scenes/Options/RecordingOptions/AmplificationSlider.cs]: When seeing the impact on the visualization, I just felt that 6 db was too much, but 0 was too little.
